visium_associated_xenium_io/download.py

```diff
@@ -1,5 +1,6 @@
 ##
```

Comment:
is this visium associated to rep1, rep2 or both? if for both, would keep it as is, otherwise would specify which rep

Reply:
associated to both